awscli/customizations/cloudformation/yamlhelper.py

```diff
@@ -80,6 +80,9 @@ def _dict_constructor(loader, node):
     return OrderedDict(loader.construct_pairs(node))
 
 
+class SafeLoaderWrapper(yaml.SafeLoader):  # type: ignore
```

Comment:
Let's add some comment do we need this because it's not clear without full context.

Reply:
And let's remove "# type: ignore" because we don't check typing anyway

Reply:
Done!